src/lib/components/workflow/workflows-summary-configurable-table/table-empty-state.svelte

```diff
@@ -0,0 +1,83 @@
+<script>
+  import { page } from '$app/stores';
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { workflowError } from '$lib/stores/workflows';
+  import emptyImage from '$lib/vendor/empty-state-dark_2x.png';
+  import noResultsImages from '$lib/vendor/empty-state-light_2x.png';
+
+  $: query = $page.url.searchParams.get('query');
+
+  const samples = [
+    'samples-go',
+    'samples-java',
+    'samples-typescript',
+    'samples-python',
+    'samples-dotnet',
+    'samples-php',
+  ];
+</script>
+
+<div
+  class="flex flex-col xl:flex-row w-full h-auto rounded-bl-xl rounded-br-xl border-2 border-gray-900 overflow-hidden"
```

Comment:
Looks like there is an unnecessary border on the top here  

```suggestion
  class="flex flex-col xl:flex-row w-full h-auto rounded-bl-xl rounded-br-xl border-t-0 border-2 border-gray-900 overflow-hidden"
```

And also some responsive wonkiness (but I think that is existing behavior)

<img width="555" alt="Screenshot 2023-08-09 at 3 17 00 PM" src="https://github.com/temporalio/ui/assets/15069288/f9b0915f-bf2b-406e-97a5-50af9be6f196">

Reply:
Background color for the win this morning

<img width="590" alt="Screen Shot 2023-08-10 at 9 05 00 AM" src="https://github.com/temporalio/ui/assets/7967403/e373dc60-ba32-473f-bfe1-d0035976d55a">